vcr/cassette.py

```diff
@@ -123,25 +127,7 @@ def handle_function(cassette):
 
         return self._handle_function(fn=handle_function)
 
-    def _handle_generator(self, fn):
```

Comment:
what does `_loc` mean?

Reply:
also no need to expose this as a method - follow handle_coroutine's implementation